### PR TITLE
chore: update demo images

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by bstasyszyn — APPROVED** [review]

File: test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {
   "genesisTime": 1,
   "multihashAlgorithms": [18],
-  "maxOperationCount": 5000,
+  "maxOperationCount": 10,
```
Comment (sandrask) [inline test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json:4]: Why this change?

Comment (fqutishat) [reply test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json:4]: copy from sidetree-fabric bddtest config

**Review by rolsonquadras — APPROVED** [review]

File: test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json (hunk 1 above)
Comment (troyronda) [reply test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json:4]: Let's not change it to a small number. @fqutishat

Comment (troyronda) [reply test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json:4]: also looks like the filename didn't get renamed from 0_1 @sandrask

Comment (sandrask) [reply test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json]: What version would you like 1_0?

Comment (sandrask) [reply test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json]: @fqutishat We also need to change sidetree/0.0.1 to sidetree/v1 in config files

Comment (fqutishat) [reply test/bdd/fixtures/fabric/config/ledger/did-sidetree-protocol-v0_1.json]: done

**Review by sandrask — APPROVED** [review]
